extensions/wi-fi/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Wi-Fi Changelog
 
+## [Add Toggle Wi-Fi command] - {PR_MERGE_DATE}
```

Comment:
sorry for the nitpicking 🙈  but for the sake of not being repetitive in the changelog you could change the title here to "New Command", "New Feature" or "New Addition"